code/game/objects/items/weapons/manuals.dm

```diff
@@ -20,13 +20,14 @@
 		</head>
 
 		<body>
-		<iframe width='100%' height='97%' src="[config.wikiurl]Guide_to_construction&printable=yes&remove_links=1" frameborder="0" id="main_frame"></iframe>
+		<iframe width='100%' height='97%' src="http://wiki.syndi-cat.com/index.php?title=Guide_to_construction&printable=yes&remove_links=1" frameborder="0" id="main_frame"></iframe>
```

Comment:
Not a good design

Not needing to edit configs is good, but that doesn't mean full urls should be hardcoded everywhere
At the very least, wiki link should be a define
Ideally, make a proc for a wiki link that also adds stuff like the printable and remove links, so that you only need to supply an article name

Reply:
I'd need to know what the config is going to be set to, otherwise there are issues which will make the link not work. For example if the config has http://wiki.syndi-cat.com/index.php/Main_Page or http://wiki.syndi-cat.com/index.php/ set as its wikiurl, it won't pull up a page. So I'd need to know what the wikiurl is going to be set as, since it will need to be http://wiki.syndi-cat.com/index.php?title= I'm unaware if any other url works.